Gems/AtomLyIntegration/CommonFeatures/Assets/Scripts/Profiling/OutputProfileData.lua

```diff
@@ -100,11 +97,17 @@ function OutputProfileData:OnActivate()
         -- register for ebus callbacks
         self.tickHandler = TickBus.Connect(self)
         self.profileCaptureNotificationHandler = ProfilingCaptureNotificationBus.Connect(self)
-        self.active = true
+
+        -- output test metadata
+        ProfilingCaptureRequestBus.Broadcast.CaptureBenchmarkMetadata(self.profileName, tostring(self.outputFolder) .. "/benchmark_metadata.json")
```

Comment:
this line is too long for me to see all of it, can we reformat with line continuation?

Reply:
The line wraps for me?